src/components/AnonymousChat/ChatDetailHeader.tsx

```diff
@@ -18,13 +19,9 @@ const styles = StyleSheet.create({
     display: 'flex',
     flexDirection: 'row',
     height: 50,
-    alignItems: 'center'
-  },
-  bgDarkBlue: {
-    backgroundColor: colors.darkBlue
-  },
-  bgBondiBlue: {
-    backgroundColor: colors.bondi_blue
+    alignItems: 'center',
+    borderBottomWidth: 1,
+    borderBottomColor: colors.alto
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code removes the background color styles `bgDarkBlue` and `bgBondiBlue` from the header. This change is consistent with the PR summary, which mentions that the UI has been enhanced. However, it's important to ensure that this change doesn't negatively affect the visibility of the header in different themes or modes (light/dark) of the app.



<!-- This is an auto-generated comment by OSS CodeRabbit -->